src/main/java/pingis/controllers/MiscController.java

```diff
@@ -0,0 +1,18 @@
+package pingis.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+public class MiscController {
+
+    @RequestMapping("/")
+    public String index() {
+        return "index";
+    }
+
+    @RequestMapping("/sandbox")
```

Comment:
@dwarfcrank 
My point with this controller was to handle any request **that isn't related to any entity**. I agree it's not super relevant yet, as we even don't know what the index.html will contain in the future. Maybe its just me and my weird obsessions to separate things very early :)

@DeepIntuition 
As the purpose of the controller is to handle **any** request that isn't related to any entity, I would't use`IndexController` here. Maybe I could rename it to `GeneralController` or something similar, depending if decide to use it or not.